docs/index.md

```diff
@@ -107,8 +102,7 @@ These are courses that demonstrate how to use Nextflow features in more detail o
 
     This is a course for Nextflow developers who wish to widen their range and/or deepen their skills. Although the modules are presented linearly, learners are welcome to pick and choose topics in any order. Any dependencies on components/skills that go beyond the scope of the 'Hello Nextflow' course are indicated in the corresponding module overview.
 
-    [Launch the Side Quests training :material-arrow-right:](side_quests/index.md){ .md-button .md-button--primary }
--->
+    [Launch the Side Quests training :material-arrow-right:](side_quests/){ .md-button .md-button--primary }
```

Comment:
This page is a bit unfinished, but I think it's fine. Can maybe follow with another PR to handle that:

https://training.nextflow.io/latest/side_quests/

Reply:
Moved to https://github.com/nextflow-io/training/issues/552 so that I can resolve this and merge.